tests/src/system/basic/Swift3WhiskObjectTests.scala

```diff
@@ -50,18 +50,21 @@ class Swift3WhiskObjectTests
             }
 
             // invoke the action
-            val run = wsk.action.invoke(actionName, Map("key0" -> "value0".toJson))
+            val run = wsk.action.invoke(actionName)
             withActivation(wsk.activation, run, initialWait = 5 seconds, totalWait = 60 seconds) {
                 activation =>
-                    val logs = activation.fields("logs").toString
+                    // should be successful
+                    activation.fields("response").asJsObject.fields("success").toString should be("true")
 
-                    logs should include("It is now")
-                    logs should not include ("Could not parse date of of the response.")
-                    logs should not include ("Could not invoke date action.")
+                    // should have a field named "activationId" which is the date action's activationId
+                    activation.fields("response").asJsObject.fields("result").asJsObject.fields("activationId") should not be (null)
+
+                    // should have somewhere in the "result" field the phrase "It is now" printed from the invoked date action
+                    activation.fields("response").asJsObject.fields("result").toString should include("It is now")
```

Comment:
One thing to consider is that the result may not be present if a blocking  invoke turns into a non-blocking one.